hollow/src/main/java/com/netflix/hollow/core/validator/DuplicateDataDetectionValidator.java

```diff
@@ -0,0 +1,86 @@
+package com.netflix.hollow.core.validator;
```

Comment:
Can we put this in a package more closely aligned with the Producer API?  For example `com.netflix.hollow.api.producer.validation`.